assets/xml/objects/object_ds2n.xml

```diff
@@ -1,55 +1,83 @@
+        <Texture Name="gDs2nShoesTex" OutName="shoes" Format="rgba16" Width="16" Height="32" Offset="0x44D8" />
+
+        <!-- from 48D8 -> 4CD8 is unacounted for, it is a dark green mix of pixels -->
+            <!-- <Blob Name="gDs2n_Blob_0048D8" Size="0xC00" Offset="0x48D8" /> -->
+        <!-- These two eye textures are corrupted, they extract and draw broken in-game.
+            The half tex: the blue iris is olive greenish and whites of the eyes are blended blue, with scatter of random pixels.
+            The closed has an extra dot of black in the center of the eye lid. -->
+        <Texture Name="gDs2nEyeHalfTex" OutName="eye_half" Format="ci8" Width="32" Height="32" Offset="0x4CD8" TlutOffset="0x40D8"/>
+        <Texture Name="gDs2nEyeClosedTex" OutName="eye_closed" Format="ci8" Width="32" Height="32" Offset="0x50D8" TlutOffset="0x40D8"/>
+
+        <Texture Name="gDs2nFingersTex" OutName="fingers" Format="ci8" Width="16" Height="16" Offset="0x54D8" TlutOffset="0x3ED8" />
+        <Texture Name="gDs2nSkinGradientTex" OutName="skin_gradient" Format="ci8" Width="8" Height="8" Offset="0x55D8" />
+        <Texture Name="gDs2nHairTex" OutName="hair" Format="rgba16" Width="16" Height="16" Offset="0x5618" />
+        <Texture Name="gDs2nHairlineTex" OutName="hairline" Format="ci8" Width="16" Height="16" Offset="0x5818" />
+        <Texture Name="gDs2nShirtButtonsTex" OutName="shirt_buttons" Format="ci8" Width="16" Height="32" Offset="0x5918" />
+        <Texture Name="gDs2nVestTex" OutName="vest" Format="rgba16" Width="16" Height="16" Offset="0x5B18" />
+        <Texture Name="gDs2nShirtTex" OutName="shirt" Format="ci8" Width="16" Height="16" Offset="0x5D18" />
+        <Texture Name="gDs2nEyeOpenTex" OutName="eye_open" Format="ci8" Width="32" Height="32" Offset="0x5E18" TlutOffset="0x40D8" />
+
+        <Texture Name="gDs2nMouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="16" Offset="0x6218" TlutOffset="0x3CD8" />
+
+        <!-- This blob is not fully mapped  <Blob Name="gDs2n_Blob_006418" Size="0x1B78" Offset="0x6418" /> -->
+        <!-- z64Utils thinks this might be an animation, but crashes on viewing. Unused. -->
+        <!-- <Animation Name="gDs2nBrokenAnim" Offset="0x6434" /> -->
+
+        <!-- Unused and referenced. Exact copies of the above used textures. -->
+        <Texture Name="gDs2nFingers2Tex" OutName="fingers2" Format="ci8" Width="16" Height="16" Offset="0x7050" TlutOffset="0x3ED8" />
+        <Texture Name="gDs2nEyeOpen2Tex" OutName="eye_open2" Format="ci8" Width="32" Height="32" Offset="0x7990" TlutOffset="0x40D8" />
+        <Texture Name="gDs2nMouthOpen2Tex" OutName="mouth_open2" Format="ci8" Width="32" Height="16" Offset="0x7D90" TlutOffset="0x3CD8" />
+
+        <Animation Name="gDs2nIdleAnim" Offset="0x8038" />
+
+        <!-- z64Utils cannot render this skeleton, the only limb I can identify is head from headtracking function -->
```

Comment:
If your interested, I have a method to help determine the limbs without z64Utils that I used to determine the limbs for Cne01.